### PR TITLE
Add studies for prod84

### PR DESCRIPTION
**Pull request by dominikl**
Add the stats for the three new releases for prod84.

### CODE REVIEW

File: _data/releases.tsv [hunk 1]
```diff
@@ -37,3 +37,4 @@ Date	Data release	Code version	Sets	Wells	Experiments	Images	Planes	Size (TB)	Fi
 2020-04-27	prod81	0.8.1	12579	1195994	1163031	9084406	63757664	169.25	23.18	366
 2020-05-19	prod82	0.8.2	12617	1210586	1177623	9098998	63786848	169.48	23.22	367
 2020-06-15	prod83	0.8.3	12617	1210586	1177623	9098998	63786848	169.48	23.22	367
+2020-06-30	prod84	0.8.4	12935	1210586	1178081	9104040	64886054	180.8749	23.23
```
Comment (sbesson) [inline _data/releases.tsv:40]: Database size is `358G` (probably some shrinkage due to the PSGL 11 dump/restore process ?)

File: _data/studies.tsv [hunk 2]
```diff
@@ -75,8 +75,12 @@ idr0066-voigt-mesospim	experimentC	prod70	853	3	0	3		0	4	3739	0.03797700596	3797
 idr0066-voigt-mesospim	experimentD	prod70	854	1	0	1		0	2	2978	0.04118653215	41186532145	6	6864.422024	5025 x 6735 x 1489 x 1 x 1
 idr0067-king-yeastmeiosis	experimentA	prod72	904	60	0	60		0	419	136360	0.2854708875	285470887482	1594	179.0908955	1224 x 1224 x 8 x 2 x 36
 idr0069-caldera-perturbome	screenA	prod71	2251	124	47616	47616			190464	571392	1.225666503	1225666502656	571912	2.143103314	1360 x 1024 x 1 x 3 x 1
+idr0070-kerwin-hdbr	experimentA	prod84	1104	287	0	287		0	4554	13512	4.9	1.33536E+11	9219	14484878.44	8223 x 9270 x 1 x 3 x 1
```
Comment (sbesson) [inline _data/studies.tsv:78]: `Size (TB)` vs `Size` do not seem to match for most studies ?

File: _data/studies.tsv [hunk 3]
```diff
@@ -75,8 +75,12 @@ idr0066-voigt-mesospim	experimentC	prod70	853	3	0	3		0	4	3739	0.03797700596	3797
...
 idr0073-schaadt-immuneinfiltrates	experimentA	prod80	1002	193	0	3			193	579	0.02230089	24520087958	580	121.1615447	5578 x 5048 x 1 x 3 x 1
 idr0075-cabirol-honeybee	experimentA	prod73	951	10	0	10			20	1932	0.0009225002	1014299685	81	48.3655779362	512 x 512 x 100 x 1 x 1
+idr0077-valuchova-flowerlightsheet	experimentA	prod84	1101	9	0	149		0	76	1005278	6.3	6.49073E+12	1938	3349191015	 1780 x 1780 x 99 x 4 x 32
```
Comment (sbesson) [inline _data/studies.tsv:81]: Also the Average size (`14484878.44`) looks to be in `B` ?

File: _data/studies.tsv (hunk 2 above)
Comment (dominikl) [reply _data/studies.tsv:78]: Well, the reason for this is:
```
$ /opt/omero/server/OMERO.server/bin/omero fs usage Project:1104
Total disk usage: 133537132989 bytes in 9220 files
```
But:
```
$ python idr-utils/scripts/stats.py
...
 idr0070-kerwin-hdbr/experimentA                  | 1104    | 287 of 2362    | 0       | 4554    | 13512    | 4.9 TB
...
```

Comment (sbesson) [reply _data/studies.tsv:78]: Ah thanks. This comes back to which metric we want to use. I think the difference is that:

- `omero fs usage` is reporting the size of files on disk 
- the `stats.py` is only querying the DB and computing the bytes size from the image dimensions as well as the pixel type

The major difference between the two of them probably reflects file compression esp. for large volumes.

If we collectively agree the first metric should be used, I'll propose to open a PR against idr-utils to remove `Bytes` to clear the ambiguity.

**Comment by sbesson** [comment]
Changes look good. There is still some ambiguity about the `Size` that should be used but this should not hold the release of 3 studies. Merging and we can rationalize.

**Pull request merged**